crud.py

```diff
@@ -0,0 +1,53 @@
+from typing import Optional
+
+from sqlalchemy.orm import Session
+
+import models
+import schemas
+
+
+def get_all_author(db: Session, skip: int = 0, limit: int = 10):
+    return db.query(models.DBAuthor).offset(skip).limit(limit).all()
+
+
+def get_author_by_name(db: Session, name):
```

Comment:
Consider adding a type annotation for the `name` parameter in the `get_author_by_name` function to improve code readability and type checking.